packages/test/harness/src/host/dev/hostOverrides/error.js

```diff
@@ -1,12 +1,11 @@
 // In dev mode, draw a red cross when test failed.
 
 const dumpLogs = require('../../common/dumpLogs');
-const override = require('../utils/override');
 const stripANSI = require('strip-ansi');
 
 // Send the error back to the browser console.
-module.exports = (webDriver, error) =>
-  override(error, async function error(error) {
+module.exports = webDriver =>
+  async function error(error) {
```

Comment:
Are there rules for Promise vs async/await?

Reply:
I guess you mean, `then`/`catch` vs. `async`/`await`.

99% of the time, use `async`/`await`.

1%, IMO, is the main entrypoint:

```js
async function main() {
  // This is main entrypoint
}

// You can't use await at root level
main().catch(err => {
  console.error(err);
  process.exit(-1);
});
```